### DISCUSSION TITLE
[Feature Request] More than 1 minute to shake to restart sleep timer (changing code from '1.minutes' to '30.minutes' doesn't work)

### DISCUSSION THREAD

**Discussion by SantoshGuptaML** [PaulWoitaschek/Voice#1374] [Ideas]
Sometimes after an audiobook as finished playing I want to lay in bed for 5-10 minutes to see if I fall asleep, and if not, I want to shake to restart. 

I believe at the moment you need to shake within 30 seconds to restart. I've tried after a minute and it doesn't work.

**Comment by SantoshGuptaML** [comment]
It seems that there might be a system limit or something for increasing the time

https://github.com/PaulWoitaschek/Voice/blob/5.0.2-2/app/src/main/java/de/ph1b/audiobook/playback/SleepTimer.kt#L66

I set this to more than 30 seconds, but it seems that if I don't shake within 30 seconds if the display is turned off, it doesn't restart.

**Comment by SantoshGuptaML** [comment]
I'm looking at v.5.0.1 now. This one seems to last 1 minute after play back, 

https://github.com/PaulWoitaschek/Voice/blob/5.0.1/app/src/main/java/de/ph1b/audiobook/playback/SleepTimer.kt#L72

I changed it to 30.minutes but somehow it doesn't work after 30 minutes. The shake needs to happen within 1 minute.

**Comment by bjohas** [comment]
More flexibility to restart would be nice - also see https://github.com/PaulWoitaschek/Voice/discussions/1288.
